liblustreapi-sys/build.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+fn main() {
+    let out_file = std::env::current_dir().unwrap().join("src").join("bindings.rs");
+
+    // Tell cargo to tell rustc to link the system liblustreapi
+    // shared library.
+    println!("cargo:rustc-link-lib=lustreapi");
+
+    if out_file.exists() {
+        return;
```

Comment:
Might be useful to write information to debug indicating that the file already exists so no processing will take place.

Reply:
You can't, this is in a build script (notice the println! above).